cassandra-bridge/build.gradle

```
@@ -39,6 +39,7 @@ publishing {
 }
 
 dependencies {
+    implementation(project(':cassandra-analytics-common'))
```

We should just declare this as an API implementation so we don't need to add both the bridge and common as dependencies in other subprojects (i.e four-zero)
```suggestion
    api(project(':cassandra-analytics-common'))
```